### PR TITLE
fixed table headers added to multiple tables.

### LINKED ISSUE #458: "sticky" table header for analysis and comparison widgets
The long analysis table and long comparison table header should be sticky. 
- Building damage table & comparison
- Building functionality and MCS tables & comparison

e.g. 
Image
Image

### PR DESCRIPTION
**Pull request by Rashmil-1999**
Tables to Check would be Building Damage, Functionality, Population dislocation and their respective comparisons and Retrofit summary table.

### CODE REVIEW

File: src/components/childComponents/table/BldgDamageTable.jsx [hunk 1]
```diff
@@ -169,7 +172,12 @@ export function BldgDamageTable(props) {
 					label="View Percentage"
 				/>
 			</Box>
-			<Table className={classes.table} size="small" aria-label="a dense table">
+			<Table
```
Comment (longshuicy) [inline src/components/childComponents/table/BldgDamageTable.jsx]: When scrolling the View percentage got scrolled up and has no way to get to it. Maybe move that outside the table container.
image
image

File: src/components/childComponents/table/FunctionalityTable.jsx [hunk 2]
```diff
@@ -117,8 +120,13 @@ export function FunctionalityTable(props) {
 	const classes = useStyles();
 
 	return (
-		<TableContainer>
-			<Table className={classes.table} size="small" aria-label="a dense table">
+		<TableContainer className={classes.tableContainer}>
```
Comment (longshuicy) [inline src/components/childComponents/table/FunctionalityTable.jsx:126]: same with percent and % of buildings
image

Comment (longshuicy) [reply src/components/childComponents/table/FunctionalityTable.jsx:126]: Actually you can look at the comparison table. That seems to be correct.
image

File: src/components/childComponents/table/HuaResultsTable.jsx [hunk 3]
```diff
@@ -170,10 +173,11 @@ export function HuaResultsTable(props) {
 			</div>
 
 			{resultsJson !== undefined ? (
-				<TableContainer>
+				<TableContainer className={classes.tableContainer}>
```
Comment (longshuicy) [inline src/components/childComponents/table/HuaResultsTable.jsx:176]: Same for this table and comparison, you might want to change the header cell color to not be transparent. When scrolling some of the those become see through.

image

File: src/components/childComponents/RetrofitStrategySummary.jsx [hunk 4]
```diff
@@ -315,8 +318,12 @@ export default function RetrofitStrategySummary(props) {
...
 			</Grid>
 			<Grid item xs={12}>
-				<TableContainer>
-					<Table className={classes.table} aria-label="simple table">
+				<TableContainer className={classes.tableContainer}>
+					<Table
+						stickyHeader
+						className={classes.table}
+						aria-label="simple table"
+					>
```
Comment (longshuicy) [inline src/components/childComponents/RetrofitStrategySummary.jsx:329]: The header color disappeared for some reason. Also the border becomes so thick. Could you compare what's on production.

prod:
image


Your PR:
image

**Review by longshuicy — CHANGES_REQUESTED** [review]
Overall looks good. Just some minor suggestions for change.

File: src/components/childComponents/table/FunctionalityTable.jsx (hunk 2 above)
Comment (Rashmil-1999) [reply src/components/childComponents/table/FunctionalityTable.jsx:126]: This one was like this I didn't touch anything else. I can make it look like the compare. Do you want me to do that? That involves some changes in table cell movements and stuff..

Comment (longshuicy) [reply src/components/childComponents/table/FunctionalityTable.jsx:126]: I see. could you make them the same?

File: src/components/childComponents/RetrofitStrategySummary.jsx (hunk 4 above)
Comment (Rashmil-1999) [reply src/components/childComponents/RetrofitStrategySummary.jsx:329]: Done @longshuicy

**Review by longshuicy — APPROVED** [review]
I slightly added a bit container height for the population and functionality table. Approving.

**Pull request merged**